astropy/utils/iers/iers.py

```diff
@@ -61,22 +74,6 @@
     "IETF_LEAP_SECOND_URL",
 ]
 
-# IERS-A default file name, URL, and ReadMe with content description
-IERS_A_FILE = "finals2000A.all"
```

Comment:
This is going to break a lot of stuff. We need backward compatibility.

Reply:
These variables are imported higher up so should work